docs/deprecations.md

```diff
@@ -17,6 +17,12 @@ disabledDeprecations:
   - '*' # To disable all deprecation messages
 ```
 
+<a name="NEW_PACKAGING_PATTERNS"><div>&nbsp;</div></a>
+
+## New way to define packaging patterns
```

Comment:
Could you please add `Deprecation code:` in a similar way to all other deprecations below?